crates/cargo-test-support/src/install.rs

```diff
@@ -5,7 +5,10 @@ use std::path::{Path, PathBuf};
 /// Used by `cargo install` tests to assert an executable binary
 /// has been installed. Example usage:
 ///
-///     assert_has_installed_exe(cargo_home(), "foo");
+/// use cargo_test_support::install::assert_has_installed_exe;
+/// use cargo_test_support::install::cargo_home;
+///
+/// assert_has_installed_exe(cargo_home(), "foo");
```

Comment:
Should we use code block here?

```suggestion
/// ```no_run
/// use cargo_test_support::install::assert_has_installed_exe;
/// use cargo_test_support::install::cargo_home;
///
/// assert_has_installed_exe(cargo_home(), "foo");
```